mysql-test/suite/innodb/t/xtradb_compressed_columns_upgrade.test

```diff
@@ -0,0 +1,125 @@
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+
+# making sure that the next tablespace id and clustered index id will be exactly the
+# same as in the pre-included 5.6 files
+--let $desired_tablespace_id = 100
+--let $desired_index_id = 150
+
+--disable_query_log
+
+CREATE TABLE dummy_t (id INT) ENGINE=InnoDB;
+CREATE INDEX dummy_idx ON dummy_t(id);
+--let $initial_max_tablespace_id = `SELECT MAX(space) FROM information_schema.innodb_sys_tablespaces`
+--let $initial_max_index_id = `SELECT MAX(index_id) FROM information_schema.innodb_sys_indexes`
+
+if($initial_max_tablespace_id >= $desired_tablespace_id)
+{
+  DROP TABLE dummy_t;
+  --skip Unable to create table with desired tablespace id
```

Comment:
I see, but what I'm worried about is that after some perturbation we can arrive in the situation where all the test executions are skipped, and it would be very hard to notice. Maybe --force-restart would help here?